spec/javascript/pages/browse/ContactIcons.spec.js

```diff
@@ -6,9 +6,9 @@ describe('ContactIcons', () => {
   it('generally works', function() {
     const wrapper = shallowMount(ContactIcons, {
       localVue: configure(createLocalVue()),
-      propsData: { contactTypes: [{id: 1, name: "email"}, {id: 2, name: "text"},{id: 3, name: "call"}]}
```

Comment:
why didn't you want to use `call` anymore? 
my earlier thinking (open to being convinced otherwise!) was that if ppl say phone, they could maybe think it means text ? (harder to do tho, when the list is right there...)